specification/cognitiveservices/Language.Conversations/main.tsp

```diff
@@ -1161,9 +1107,19 @@ model ConversationSummarizationTaskParameters is PreBuiltTaskParameters {
 }
 
 /** Supported parameters for a custom conversation summarization task. */
-@added(Versions.v2023_11_15_preview)
+@added(Versions.v2024_05_15_preview)
 model CustomConversationSummarizationTaskParameters is CustomTaskParameters {
-  ...ConversationSummarizationTaskParametersBase;
+  /** It controls the approximate number of sentences in the output summaries. */
+  sentenceCount?: int32;
+
+  /** String index type */
+  stringIndexType?: StringIndexType = StringIndexType.TextElements_v8;
+
+  /** (NOTE: Recommended to use summaryLength over sentenceCount) Controls the approximate length of the output summaries. */
```

Comment:
Is this note meant for customers? If so, for docs, I'd consider reformatting this. It just reads like it was meant for internal devs.

Reply:
Combined into documentation by adding later in the same sentence.